embed/config.go

```diff
@@ -136,6 +136,13 @@ type Config struct {
 	//	}
 	//	embed.StartEtcd(cfg)
 	ServiceRegister func(*grpc.Server) `json:"-"`
+
+	// auth
+
+	TokenType      string
```

Comment:
On the second thought, I also think that "simple" would be suitable. Printing warning in the log would be enough.